### PR TITLE
[Stock-RM] Add 'New Spawn' group [POST TRAVIS]

### PR DESCRIPTION
**Pull request by darkelement1987**
## Description
There is a current issue of a brand new pokemon spawn and it will not be set a rarity until the dynamic rarity filer is updated.

We could set all new pokemon to ultra rare but for people who dont update rarity often this would mean things as common as pidgey would stay ultra rare until rarity file is next updated.

This way sets all new pokemon as a "New Spawn" and once the rarity file is updated it will be sorted properly. I added "New Spawn" to notify by rarity on the side bar and also included it in scale by rarity.

## How Has This Been Tested?
Local instance. Works fine

**My instances screenshot**
image

## Screenshots (if appropriate):

**Original PR Screenshot**
image

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: static/js/map.common.js [hunk 1]
```diff
@@ -1165,7 +1165,7 @@ function getPokemonRarity(pokemonId) {
         return i8ln(pokemonRarities[pokemonId])
     }
 
-    return ''
+    return 'New Spawn'
```
Comment (rlodev67) [inline static/js/map.common.js]: It should be i8ln('New Spawn') for non english people

Comment (darkelement1987) [reply static/js/map.common.js]: Ok will change

**Comment by darkelement1987** [comment]
@rlodev67 Reminder of the internationalisation bug, this might solve it?

@ models.py
`rarity_list = {'Common': 0, 'Uncommon': 1, 'Rare': 2, 'Very Rare': 3, 'Ultra Rare': 4, 'New Spawn': 5}`

Not sure?

**Comment by darkelement1987** [comment]
Need translations for:

China
Hong Kong
Taiwan
Japan
Russia
Korea
Portugal (think this is just the english translation but not sure)

**Pull request merged**